### ISSUE TITLE
Failed to get kernel serve port

### ISSUE THREAD

**Issue by huangkenicole** [siyuan-note/siyuan#9823]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Can the issue be reproduced with the default theme (daylight/midnight)?

- [X] I was able to reproduce the issue with the default theme

### Could the issue be due to extensions?

- [X] I've ruled out the possibility that the extension is causing the problem.

### Describe the problem

获取内核服务端口失败，请确保程序拥有网络权限并不受防火墙和杀毒软件阻止。
Failed to get kernel serve port, please make sure the program has network permissions and is not blocked by firewalls and antivirus software.

### Expected result

can't open the app on the mac(apple chip)

### Screenshot or screen recording presentation

<img width="701" alt="image" src="https://github.com/siyuan-note/siyuan/assets/112435311/0b657966-f7d4-4663-b04c-16f5e2b681cd">

### Version environment

```markdown
- Version: 
- Operating System: 
- Browser (if used):
```

### Log file

none

**Comment by 88250** [comment]
Please upgrade to the latest version, close the SiYuan-Kernel process in the monitor, and then start it again.

**Comment by huangkenicole** [comment]
this version is the latest, after I updated to the latest version, it exists this problem.
and I have kill all the process that relevant to siyuan, but it still doesn't work

**Comment by 88250** [comment]
Please upload workspace/temp/siyuan.log

**Comment by huangkenicole** [comment]
here you go
![image](https://github.com/siyuan-note/siyuan/assets/112435311/71e9040b-4c22-46f3-8b63-d89d6260841b)

I have tried the old version, but it still has same problem, and I restart my laptop, but it still can't open.

my chip is Mac Apple M1, I have solved now but I didn't know the reason. (I re-downloaded the Intel version of the dmg file, and now I am able to reopen it successfully, but I don't know whether I lost some text ). Many thanks.
